### PR TITLE
Fix cluster manager with LocalNet on Helm charts

### PR DESCRIPTION
**Pull request by okdas**
## Description

After we switched to Helm charts on LocalNet, the cluster manager stopped automatically staking validators beyond four pre-staked validators.

## Type of change

Please mark the relevant option(s):

- [x] Bug fix

## List of changes

- Change the way cluster manager looks up validator to stake automatically.

## Testing

- [ ] `make develop_test`; if any code changes were made
- [ ] [Docker Compose LocalNet](https://github.com/pokt-network/pocket/blob/main/docs/development/README.md); if any major functionality was changed or introduced
- [x] [k8s LocalNet](https://github.com/pokt-network/pocket/blob/main/build/localnet/README.md); if any infrastructure or configuration changes were made

## Required Checklist

- [ ] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have added, or updated, [`godoc` format comments](https://go.dev/blog/godoc) on touched members (see: [tip.golang.org/doc/comment](https://tip.golang.org/doc/comment))
- [ ] I have tested my changes using the available tooling
- [ ] I have updated the corresponding CHANGELOG

### If Applicable Checklist

- [ ] I have updated the corresponding README(s); local and/or global
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] I have added, or updated, [mermaid.js](https://mermaid-js.github.io) diagrams in the corresponding README(s)
- [ ] I have added, or updated, documentation and [mermaid.js](https://mermaid-js.github.io) diagrams in `shared/docs/*` if I updated `shared/*`README(s)

### CODE REVIEW

File: build/localnet/cluster-manager/main.go [hunk 1]
```diff
@@ -33,7 +33,7 @@ var (
 )
 
 func init() {
-	rpcURL = fmt.Sprintf("http://%s:%s", runtime.GetEnv("RPC_HOST", "v1-validator001"), defaults.DefaultRPCPort)
+	rpcURL = fmt.Sprintf("http://%s:%s", runtime.GetEnv("RPC_HOST", "validator-001-pocket-validator"), defaults.DefaultRPCPort)
```
Comment (Olshansk) [inline build/localnet/cluster-manager/main.go:36]: Use the `Validator1EndpointK8S` constant you defined elsewhere.

File: build/localnet/cluster-manager/main.go [hunk 2]
```diff
@@ -77,7 +77,7 @@ func main() {
 				continue
 			}
 
-			validatorServiceUrl := fmt.Sprintf("v1-validator%s:%d", validatorId, defaults.DefaultP2PPort)
+			validatorServiceUrl := fmt.Sprintf("validator-%s-pocket-validator:%d", validatorId, defaults.DefaultP2PPort)
```
Comment (Olshansk) [inline build/localnet/cluster-manager/main.go:80]: NIT: Create a local constant like `const validatorServiceURLFormat = "validator-%s-pocket-validator:%d"`

**Review by Olshansk — CHANGES_REQUESTED** [review]

File: build/localnet/cluster-manager/main.go (hunk 1 above)
Comment (okdas) [reply build/localnet/cluster-manager/main.go:36]: Great idea, especially with `defaults` already imported.

**Comment by Olshansk** [comment]
> E2E tests fail with a `panic: secrets "v1-localnet-validators-private-keys" not found` error on my local machine. Requesting changes to get them passing again.

Which command did you run for the E2E test?

**Review by Olshansk — CHANGES_REQUESTED** [review]
Code LGTM but dittoing @dylanlott's comment. 

Looking forward to having E2E part of our CI/CD branch protection.

**Comment by okdas** [comment]
I'm going to close this pull request and I moved the changes to #674 because these values need to be adjusted in the next PR anyway, and that next PR is almost ready for review.

**Pull request closed without merging**